lib/chef-cli/cookbook_profiler/git.rb

```diff
@@ -103,6 +104,17 @@ def current_branch
       end
 
       private
+      
+      def repo_directory
+        @repo_directory ||=
+          Pathname.new(@cookbook_path).ascend do |parent_dir|
+            possbile_git_dir = File.join(parent_dir, '.git')
+            if File.directory?(possbile_git_dir)
+              return possbile_git_dir
```

Comment:
Don't use return from a block, it bypass everything (especially the `@xxx ||=` )

Reply:
Done